lib/timeline/component/item/Item.js

```diff
@@ -83,6 +90,21 @@ class Item {
     if (this.displayed) this.redraw();
   }
 
+  /**
+   * Set whether the item can be selected.
+   * Can only be set/unset if the timeline's `selectable` configuration option is `true`.
+   * @param {Object} data `data` from `constructor` and `setData`
+   */
+  setSelectability(data) {
+    if (data) {
+      if (data.selectable !== undefined) {
```

Comment:
`data.selectable !== undefined` will also allow `null` to  act as true.
Maybe doing something like this is better:
``` javascript
 this.selectable = typeof data.selectable === 'undefined' ? true : Boolean(data.selectable)
```

Reply:
Will do.